docs/docs/references/users/README.md

```diff
@@ -1,5 +1,95 @@
 # 👨‍👩‍👧‍👦 Users
 
+The _users_ are the main entities of the identity service.
+We will describe the user-related concepts and details in the following.
+
 ## Profile
 
+Each user has a profile containing the user information, more than the name and contact details.
+
+A user profile consists of 3 sections:
+
+- **Basic data**: contains basic info in pre-defined attributes, such as user id, username, email, phone number, and when the user last signed in.
+- **Custom data**: contains additional info in customizable attributes, such as user-preferred color and language.
+- **Identities**: contains the user info retrieved from social sign-in, such as Facebook, GitHub, and WeChat.
+
+### All attributes
+
+The following attributes (except password-related) are visible on the user profile.
```

Comment:
Feels like we do not have the concept of the user profile invisible/visible.

Reply:
Updated. PTAL